src/test/ui/async-await/async-borrowck-escaping-block-error.stderr

```diff
@@ -0,0 +1,22 @@
+error[E0373]: closure may outlive the current function, but it borrows `x`, which is owned by the current function
```

Comment:
Looks like the error is still referencing a "closure" here, which is unfortunate.

Reply:
@cramertj oops, my bad. will repair this soon.
but I'm not sure, as the [rfc](https://github.com/rust-lang/rfcs/blob/master/text/2394-async_await.md#async-blocks) said async block is equivalent to  immediately-invoked async closure.
would you mind suggesting a precise word here? cc @estebank

Reply:
"`async block` may outlive the current function"